.github/workflows/Test.yml

```diff
@@ -29,4 +29,10 @@ jobs:
       - uses: julia-actions/julia-buildpkg@v1
       - uses: julia-actions/julia-runtest@v1
         # with:
-        #   annotate: true
\ No newline at end of file
+        #   annotate: true
+      - uses: julia-actions/julia-processcoverage@v1
+      - uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }} # required
```

Comment:
The token should be added to the GH secrets (it still works on the PR without it since the PR is from a fork): https://docs.codecov.com/docs/adding-the-codecov-token